resources/views/components/section-title.blade.php

```diff
@@ -1,5 +1,5 @@
 <div class="md:col-span-1 flex justify-between">
-    <div class="px-4 sm:px-0">
+    <div class="flex-grow px-4 sm:px-0">
```

Comment:
Isn't the Tailwind v3 equivalent simply `grow`?

Reply:
My bad.